cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -336,21 +364,24 @@ install() {
         _env_script_path="${{ install_path.env_key }}{% if install_path.subdir | length %}/{% endif %}{{ install_path.subdir }}/env"
         _install_dir_expr="$_install_dir"
         _env_script_path_expr="$_env_script_path"
+
+        # Override if necessary
+        if [ -n "${CARGO_DIST_FORCE_INSTALL_DIR:-}" ]; then
+            _install_dir="$CARGO_DIST_FORCE_INSTALL_DIR"
+            # Note that it's possible for the `env` file to have
+            # been configured to a totally different path, but we don't
+            # (yet) have the appropriate value to force that to here, so
+            # we use this instead.
+            _env_script_path="$CARGO_DIST_FORCE_INSTALL_DIR/env"
```

Comment:
Could you elaborate? the previous lines seem to always put these together

Reply:
I think you're right, I was misreading the condition and thought it branched further. I'll remove the comment.